docs/pages/bare/exploring-bare-workflow.md

```diff
@@ -4,28 +4,20 @@ sidebar_title: Walkthrough
 ---
```

Comment:
this page is mostly video and is out of date, so we should just delete it because i don't think anyone plans to updte it soon.